ibpc_tester/parts_in_search_phase1.json

```diff
@@ -0,0 +1 @@
0, 18], "36": [0, 18], "37": [0, 18], "38": [0, 18], "39": [0, 18], "40": [0, 18], "41": [0, 18], "42": [0, 18], "43": [0, 18], "44": [0, 18], "45": [0, 18], "46": [0, 18], "47": [0, 18], "48": [0, 18], "49": [0, 18], "50": [0, 18], "51": [0, 18], "52": [0, 18], "53": [0, 18], "54": [0, 18], "55": [0, 18], "56": [0, 18], "57": [0, 18], "58": [0, 18], "59": [0, 18], "60": [0, 18], "61": [0, 18], "62": [0, 18], "63": [0, 18], "64": [0, 18], "65": [0, 18], "66": [0, 18], "67": [0, 18], "68": [0, 18], "69": [0, 18], "70": [0, 18], "71": [0, 18], "72": [0, 18], "73": [0, 18], "74": [0, 18], "75": [0, 18], "76": [0, 18], "77": [0, 18], "78": [0, 18], "79": [0, 18], "80": [0, 18], "81": [0, 18], "82": [0, 18], "83": [0, 18]}, "000013": {"0": [4], "1": [4], "2": [4], "3": [4], "4": [4], "5": [4], "6": [4], "7": [4], "8": [4], "9": [4], "10": [4], "11": [4], "12": [4], "13": [4], "14": [4], "15": [4], "16": [4], "17": [4], "18": [4], "19": [4], "20": [4], "21": [4], "22": [4], "23": [4], "24": [4], "25": [4], "26": [4], "27": [4], "28": [4], "29": [4], "30": [4], "31": [4], "32": [4], "33": [4], "34": [4], "35": [4], "36": [4], "37": [4], "38": [4], "39": [4], "40": [4], "41": [4], "42": [4], "43": [4], "44": [4], "45": [4], "46": [4], "47": [4], "48": [4], "49": [4], "50": [4], "51": [4], "52": [4], "53": [4], "54": [4], "55": [4], "56": [4], "57": [4], "58": [4], "59": [4], "60": [4], "61": [4], "62": [4], "63": [4], "64": [4], "65": [4], "66": [4], "67": [4], "68": [4], "69": [4], "70": [4], "71": [4], "72": [4], "73": [4], "74": [4], "75": [4], "76": [4], "77": [4], "78": [4], "79": [4], "80": [4], "81": [4], "82": [4], "83": [4]}, "000014": {"0": [19, 14], "1": [19, 14], "2": [19, 14], "3": [19, 14], "4": [19, 14], "5": [19, 14], "6": [19, 14], "7": [19, 14], "8": [19, 14], "9": [19, 14], "10": [19, 14], "11": [19, 14], "12": [19, 14], "13": [19, 14], "14": [19, 14], "15": [19, 14], "16": [19, 14], "17": [19, 14], "18": [19, 14], "19": [19, 14], "20": [19, 14], "21": [19, 14], "22": [19, 14], "23": [19, 14], "24": [19, 14], "25": [19, 14], "26": [19, 14], "27": [19, 14], "28": [19, 14], "29": [19, 14], "30": [19, 14], "31": [19, 14], "32": [19, 14], "33": [19, 14], "34": [19, 14], "35": [19, 14], "36": [19, 14], "37": [19, 14], "38": [19, 14], "39": [19, 14], "40": [19, 14], "41": [19, 14], "42": [19, 14], "43": [19, 14], "44": [19, 14], "45": [19, 14], "46": [19, 14], "47": [19, 14], "48": [19, 14], "49": [19, 14], "50": [19, 14], "51": [19, 14], "52": [19, 14], "53": [19, 14], "54": [19, 14], "55": [19, 14], "56": [19, 14], "57": [19, 14], "58": [19, 14], "59": [19, 14], "60": [19, 14], "61": [19, 14], "62": [19, 14], "63": [19, 14], "64": [19, 14], "65": [19, 14], "66": [19, 14], "67": [19, 14], "68": [19, 14], "69": [19, 14], "70": [19, 14], "71": [19, 14], "72": [19, 14], "73": [19, 14], "74": [19, 14], "75": [19, 14], "76": [19, 14], "77": [19, 14], "78": [19, 14], "79": [19, 14], "80": [19, 14], "81": [19, 14], "82": [19, 14], "83": [19, 14]}}
```

Comment:
I assume img_ids are strings here because json keys must be strings. We could instead map the scene id to an array of img_id and object_ids. But not a blocker.